test/sanity/run-test.sh

```diff
@@ -30,13 +30,21 @@ nodeid='CSINode'
 if [[ "$#" -gt 0 ]] && [[ -n "$1" ]]; then
   nodeid="$1"
 fi
+testvolumeparameters='$GOPATH/src/sigs.k8s.io/azurefile-csi-driver/test/sanity/vhd.yaml'
 
 _output/azurefileplugin --endpoint "$endpoint" --nodeid "$nodeid" -v=5 &
 
 echo 'Begin to run sanity test...'
 readonly CSI_SANITY_BIN='csi-test/cmd/csi-sanity/csi-sanity'
 if [[ "$TRAVIS" == 'true' ]]; then
-  "$CSI_SANITY_BIN" --ginkgo.v --ginkgo.noColor --csi.endpoint="$endpoint"  --ginkgo.focus='should work'
+  "$CSI_SANITY_BIN" --ginkgo.v --ginkgo.noColor --csi.endpoint="$endpoint" --ginkgo.skip='should work'
 else
-  "$CSI_SANITY_BIN" --ginkgo.v --ginkgo.noColor --csi.endpoint="$endpoint"  --ginkgo.skip='should fail when the volume source snapshot is not found|should work'
+  "$CSI_SANITY_BIN" --ginkgo.v --ginkgo.noColor --csi.endpoint="$endpoint" --ginkgo.skip='should fail when the volume source snapshot is not found|should work'
+fi
+
+echo 'Begin to run sanity test for vhd disk feature...'
+if [[ "$TRAVIS" == 'true' ]]; then
+  "$CSI_SANITY_BIN" --ginkgo.v --ginkgo.noColor --csi.endpoint="$endpoint" --csi.testvolumeparameters="$testvolumeparameters" --ginkgo.skip='should work'
+else
+  "$CSI_SANITY_BIN" --ginkgo.v --ginkgo.noColor --csi.endpoint="$endpoint" --csi.testvolumeparameters="$testvolumeparameters" --ginkgo.skip='should fail when the volume source snapshot is not found|should work'
```

Comment:
does it support non-file parameter?

Reply:
No, it just supports the parameter file.

Reply:
I think you could use shell to create a dyanmic file in sanity testing

Reply:
Ref: https://github.com/kubernetes-csi/csi-test/blob/master/cmd/csi-sanity/main.go#L85